CHANGELOG.md

```diff
@@ -1,18 +1,127 @@
+- Tracked package versions are now not required, but preferred. (#160) @robertmaynard
+- cpm_thrust would fail when provided only an install export set (#155) @robertmaynard
+- rapids_export generated config.cmake no longer leaks variables (#149) @robertmaynard
+
+## 📖 Documentation
+
+- Docs use intersphinx correctly to link to CMake command docs (#159) @robertmaynard
+- Example explains when you should use `rapids_find_generate_module` (#153) @robertmaynard
+- Add CMake intersphinx support (#147) @bdice
+
+## 🚀 New Features
+
+- Bump CPM 0.35 for per package CPM_DOWNLOAD controls (#158) @robertmaynard
+- Track package versions to the generated `find_dependency` calls (#156) @robertmaynard
+- Update to latest nvbench (#150) @robertmaynard
+
+## 🛠️ Improvements
 
-# rapids-cmake 22.02.00 (Date TBD)
+- Temporarily disable new `ops-bot` functionality (#170) @ajschmidt8
+- Use exact gtest version (#165) @trxcllnt
+- Add `.github/ops-bot.yaml` config file (#163) @ajschmidt8
 
-Please see https://github.com/rapidsai/rapids-cmake/releases/tag/v22.02.00a for the latest changes to this development branch.
+# rapids-cmake 22.02.00 (2 Feb 2022)
 
-# rapids-cmake 21.12.00 (Date TBD)
+## 🐛 Bug Fixes
+
+- Ensure that nvbench doesn&#39;t require nvml when `CUDA::nvml` doesn&#39;t exist (#146) @robertmaynard
+- rapids_cpm_libcudacxx handle CPM already finding libcudacxx before being called (#130) @robertmaynard
+
+## 📖 Documentation
+
+- Fix typos (#142) @ajschmidt8
+- Fix type-o in docs `&lt;PackageName&gt;_BINARY_DIR` instead of `&lt;PackageName&gt;_BINAR_DIR` (#140) @dagardner-nv
+- Set the `always_download` value in versions.json to the common case (#135) @robertmaynard
+- Update Changelog to capture all 21.08 and 21.10 changes (#134) @robertmaynard
+- Correct minor formatting issues (#132) @robertmaynard
+- Document how to control the git rep/tag that RAPIDS.cmake uses (#131) @robertmaynard
+
+## 🚀 New Features
+
+- rapids-cmake now supports an empty package entry in the override file (#145) @robertmaynard
+- Update NVBench for 22.02 to be the latest version (#144) @robertmaynard
+- Update rapids-cmake packages to libcudacxx 1.7 (#143) @robertmaynard
+- Update rapids-cmake packages to Thrust 1.15 (#138) @robertmaynard
+- add exclude_from_all flag to version.json (#137) @robertmaynard
+- Add `PREFIX` option to write_version_file / write_git_revision_file (#118) @robertmaynard
+
+## 🛠️ Improvements
+
+- Remove rapids_cmake_install_lib_dir unstable side effect checks (#136) @robertmaynard
+
+# rapids-cmake 21.12.00 (9 Dec 2021)
+
+## 🐛 Bug Fixes
+
+- rapids_cpm_libcudacxx install logic is safe for multiple inclusion (#124) @robertmaynard
+- rapids_cpm_libcudacxx ensures CMAKE_INSTALL_INCLUDEDIR exists (#122) @robertmaynard
+- rapids_cpm_find restores CPM variables when project was already added (#121) @robertmaynard
+- rapids_cpm_thrust doesn&#39;t place temp file in a searched location (#120) @robertmaynard
```

Comment:
FYI @ajschmidt8 the changelog generator script might be a little overzealous about HTML entity encoding. All instances of `&#39;` can be replaced with the default quote (or even a smart quote, Markdown won't care). Likewise for `&lt;` and `&gt;` for `<` and `>`. No encoding should be needed for this - or perhaps we need a step for _decoding_ entities.